src/main/java/com/mraof/minestuck/skaianet/SburbPlayerData.java

```diff
@@ -0,0 +1,274 @@
+package com.mraof.minestuck.skaianet;
+
+import com.mraof.minestuck.api.alchemy.GristType;
+import com.mraof.minestuck.computer.editmode.DeployEntry;
+import com.mraof.minestuck.computer.editmode.EditData;
+import com.mraof.minestuck.computer.editmode.ServerEditHandler;
+import com.mraof.minestuck.item.MSItems;
+import com.mraof.minestuck.network.MSPacketHandler;
+import com.mraof.minestuck.network.computer.SkaianetInfoPacket;
+import com.mraof.minestuck.player.IdentifierHandler;
+import com.mraof.minestuck.player.PlayerIdentifier;
+import com.mraof.minestuck.util.MSNBTUtil;
+import net.minecraft.nbt.*;
+import net.minecraft.resources.ResourceKey;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.server.level.ServerLevel;
+import net.minecraft.server.level.ServerPlayer;
+import net.minecraft.util.Mth;
+import net.minecraft.world.item.Item;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import javax.annotation.Nullable;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Random;
+import java.util.Set;
+import java.util.function.Supplier;
+
+/**
+ * Contains various player-specific data that were originally stored in {@code SburbConnection}.
+ * @author kirderf1
+ */
```

Comment:
I think this javadoc does a good job of describing how it relates to the old code but doesnt help much to summarize what kind of player specific data is stored in relation to sburb

Reply:
Yeah, that's pretty much all it is. It's not much more than another group of player data alternative to that in `PlayerData`. I just didn't more the data out of `SkaianetData` for better backwards-compatibility, and didn't split up the data further out of convenience.

I'm not sure if anything more can be said in the javadoc right now. I do see this as somewhat temporary though, since I've been thinking of breaking this apart using neoforge data attachments much like what I suggested for `PlayerData` in the discord.